# Request
float64 rshoulder
float64 rshoulderRot
float64 rbicep
float64 relbow
float64 rwrist 
float64 rthumb
float64 rpind 
float64 rmaj
float64 rarc
float64 rring
float64 lshoulder
float64 lshoulderRot
float64 lbicep
float64 lelbow
float64 lwrist 
float64 lthumb
float64 lpind 
float64 lmaj
float64 lring
float64 larc
float64 headPitch
float64 headYaw
---
# Response
float64 rshoulder
float64 rshoulderRot
float64 rbicep
float64 relbow
float64 rwrist 
float64 rthumb
float64 rpind 
float64 rmaj
float64 rarc
float64 rring
float64 lshoulder
float64 lshoulderRot
float64 lbicep
float64 lelbow
float64 lwrist 
float64 lthumb
float64 lpind 
float64 lmaj
float64 lring
float64 larc
float64 headPitch
float64 headYaw
